ui/component/channelStakedIndicator/view.jsx

```diff
@@ -28,34 +29,62 @@ function getChannelIcon(level: number): string {
   return icons[level] || ICONS.CHANNEL_LEVEL_1;
 }
 
+function getChannelIconB(level: number): string {
```

Comment:
White theme: Dark text on dark background.

![image](https://user-images.githubusercontent.com/64950861/149175063-01d9abc2-11c2-480a-9dda-eb4d728fcb01.png)

Reply:
Fixed. The current tooltip colors are just temporary. They used to be dynamic, but that caused some contrast errors for the tooltips in the chat for livestreams. I'll find a better solution.